openedx/core/djangoapps/dark_lang/templates/dark_lang/preview_lang_include.html

```diff
@@ -1,36 +0,0 @@
-<%!
```

Comment:
Even with a separate commit history I can't persuade GitHub to see this as the same file. The history is preserved in PyCharm though, so I think this is more of an issue with GitHub than with the Git history.